week5/5.1/I have no vinegret.py

```diff
@@ -0,0 +1,49 @@
+import time
+import random
+from datetime import datetime
+"""
+I have no vinegret!.
+By: Eliad Karni.
+
+The Project generates a date between two received dates.
+If the generated day is not a SHOPPING_DAY, it prints NO_VINEGRETE_DATE_OUTPUT message.
+
+used websites help:
+shopping day check -> https://www.delftstack.com/howto/python/python-datetime-day-of-week/
+                      https://www.geeksforgeeks.org/python-datetime-strptime-function/
+generate date -> https://stackoverflow.com/questions/553303/generate-a-random-date-between-two-other-dates
+"""
+#=============================== consts section ===============================
```

Comment:
You dont need these section comments in python - its not required, its common mostly on java from my experience

Reply:
Just replace them with two newlines - pycharm should yell at you because you are violating pep8